website/server/controllers/api-v4/user.js

```diff
@@ -244,4 +245,38 @@ api.verifyDisplayName = {
   },
 };
 
+/**
+ * @api {post} /api/v4/user/unequip/:type Unequip all items by type
+ * @apiName UserUnEquipByType
+ * @apiGroup User
+ *
+ * @apiParam (Path) {String="pet-mount-background","costume","equipped"} type The type of items
+ *                                                                       to unequip.
+ *
+ * @apiParamExample {URL} Example-URL
+ * https://habitica.com/api/v4/user/unequip/equipped
+ *
+ * @apiSuccess {Object} data user.items
+ * @apiSuccess {String} message Optional success message for unequipping an items
+ *
+ * @apiSuccessExample {json} Example return:
+ *  {
+ *   "success": true,
+ *   "data": {---DATA TRUNCATED---},
+ *   "message": "Battle Gear unequipped.
+ * }
+ *
+ */
+api.unequip = {
```

Comment:
can we have just a small integration test for this? I know it's covered in the common test but just a small one to make sure the api isn't broken

Reply:
added an integration test